pcsx2/DEV9/ATA/Commands/ATA_SCE.cpp

```diff
@@ -33,23 +44,51 @@ void ATA::HDD_SCE()
 			return;
 	}
 }
-//Has
-//ATA_SCE_IDENTIFY_DRIVE @ 0xEC
-
-//ATA_SCE_SECURITY_ERASE_PREPARE @ 0xF1
-//ATA_SCE_SECURITY_ERASE_UNIT
-//ATA_SCE_SECURITY_FREEZE_LOCK
-//ATA_SCE_SECURITY_SET_PASSWORD
-//ATA_SCE_SECURITY_UNLOCK
-
-//ATA_SCE_SECURITY_WRITE_ID @ 0x20
-//ATA_SCE_SECURITY_READ_ID @ 0x30
+// All games that have ability to install data into HDD will verify HDD by checking that this command completes successfully. Resident Evil: Outbreak for example
+// Only a few games/apps make use of the returned data, see Final Fantasy XI or the HDD Utility disks, neither of which work yet
+// Also PSX DESR bioses use this response for HDD encryption and decryption.
+// Use of external HDD ID file may be necessary for users with protected titles installed to the SCE HDD and then dumped.
+// For example: PS2 BB Navigator, PlayOnline Viewer, Bomberman Online, Nobunaga No Yabou Online, Pop'n Taisen Puzzle-Dama Online
 
+// PS2 ID Dumper can be used as test case
 void ATA::SCE_IDENTIFY_DRIVE()
 {
 	PreCmd();
 
-	//HDD_IdentifyDevice(); //Maybe?
+	// fill sceSec response with default data
+	memcpy(sceSec, "Sony Computer Entertainment Inc.", 32); // Always this magic header.
+	memcpy(sceSec + 0x20, "SCPH-20401", 10); // sometimes this matches HDD model, the rest 6 bytes filles with zeroes, or sometimes with spaces
+	memcpy(sceSec + 0x30, "  40", 4); // or " 120" for PSX DESR, reference for ps2 area size. The rest bytes filled with zeroes
+
+	sceSec[0x40] = 0; // 4-byte HDD internal SCE serial, does not match real HDD serial, currently hardcoded to 0x1000000
+	sceSec[0x41] = 0;
+	sceSec[0x42] = 0;
+	sceSec[0x43] = 0x01;
+
+	// purpose of next 12 bytes is unknown
+	sceSec[0x44] = 0; // always zero
+	sceSec[0x45] = 0; // always zero
+	sceSec[0x46] = 0x1a;
+	sceSec[0x47] = 0x01;
+	sceSec[0x48] = 0x02;
+	sceSec[0x49] = 0x20;
+	sceSec[0x4a] = 0; // always zero
+	sceSec[0x4b] = 0; // always zero
+	// next 4 bytes always these values
+	sceSec[0x4c] = 0x01;
+	sceSec[0x4d] = 0x03;
+	sceSec[0x4e] = 0x11;
+	sceSec[0x4f] = 0x01;
+	// 0x50 - 0x80 is a random unique block of data
+	// 0x80 and up - zero filled
+
+	// if exists .hddid file along with HDD image, use its content for sceSec response
+	std::string hddid_file(Path::ReplaceExtension(EmuConfig.DEV9.HddFile, "hddid"));
```

Comment:
After a quick test, I noticed you don't handle non absolute paths (i.e. the default setting of "DEV9Hdd.raw").

For Hdd images this is handled here https://github.com/PCSX2/pcsx2/blob/43f658a82bf2cbb17a5916c74a334607acc22efd/pcsx2/DEV9/DEV9.cpp#L199-L205
Where it passed to this ATA;;Open which then opens the Hdd image
https://github.com/PCSX2/pcsx2/blob/43f658a82bf2cbb17a5916c74a334607acc22efd/pcsx2/DEV9/ATA/ATA_State.cpp#L52

I would suggest against duplicating the logic I've linked, instead you could do one of following;
- Storing hddPath in the class (it currently isn't) and using it go generate hddid_file
- Generating the hddid_file using hddPath in ATA::Open and storing that for use here
- Opening the hddid file using a path based on hddPath in ATA::Open and storing the pointer for use below.

Any one of the above would allow you to handle non absolute paths.